### PR TITLE
add support to more types for json % proc

### PR DESCRIPTION
**Pull request by diegogub**
I added few more types to support. I will try to add support the unmarshal after.
Thanks

### CODE REVIEW

File: lib/pure/json.nim [hunk 1]
```diff
@@ -818,6 +818,25 @@ proc `%`*(o: enum): JsonNode =
   ## string. Creates a new ``JString JsonNode``.
   result = %($o)
 
+proc `%`*[T](t :TableRef[string,T]) : JsonNode =
```
Comment (Yardanico) [inline lib/pure/json.nim]: can you fix the code style? it should be like `t: TableRef[string, T]): JsonNode`

File: lib/pure/json.nim [hunk 2]
```diff
@@ -818,6 +818,25 @@ proc `%`*(o: enum): JsonNode =
...
+  result = newJObject()
+  for k , s in t:
```
Comment (Yardanico) [inline lib/pure/json.nim]: `k, s`

File: lib/pure/json.nim (hunk 1 above)
Comment (Yardanico) [reply lib/pure/json.nim]: and also in other places :)

Comment (diegogub) [reply lib/pure/json.nim]: sure, is there any tool for style fix?

**Comment by GULPF** [comment]
See #7203

**Comment by diegogub** [comment]
@GULPF  then I will close it. thanks

**Pull request closed without merging**